### DISCUSSION TITLE
如何使用

### DISCUSSION THREAD

**Discussion by mak589** [khoj-ai/khoj#753] [Q&A]
如何部署，怎样部署，在哪里部署！

**Comment by debanjum** [comment]
Hi @mak589, please refer to [the documentation](https://docs.khoj.dev/get-started/setup) and let us know if you're facing any specific issues with setting up Khoj?

What is your use-case for deploying Khoj? You can deploy Khoj on your own machine [with pip](https://docs.khoj.dev/get-started/setup/#installation-option-2-local) or on any cloud service [with Docker](https://docs.khoj.dev/get-started/setup/#installation-option-1-docker)?

P.S: Converting this into a discussion for now
